README.md

```diff
@@ -62,15 +62,7 @@ Or add to you pom.xml directly:
 
 ## Usage
 
-Now that you configured your POM to use the service, now you need to tell your dev mode to send emails to Mailpit by modifying the following properties in your `application.properties`:
-
-```properties
-%dev.quarkus.mailer.host=localhost
-%dev.quarkus.mailer.port=1025
-%dev.quarkus.mailer.mock=false # In dev mode, prevent from using the mock SMTP server
```

Comment:
Won't we still need `%dev.quarkus.mailer.mock=false # In dev mode, prevent from using the mock SMTP server` or else it will use Quarkus built in mock mail?

Reply:
automatically set to false if the extension is present: https://github.com/quarkiverse/quarkus-mailpit/pull/44/files#diff-589a2711c1886f50a8e49073b699d771e4039cdf449d289701aa2c3a6b40faefR105

Reply:
Nevermind I see you set it automatically in the Exposed Config.